src/bika/lims/browser/worksheet/views/results.py

```diff
@@ -49,29 +50,30 @@ def __init__(self, context, request):
             "/++plone++senaite.core.static/assets/icons/worksheet.svg"
         )
 
-        self.layout_displaylist = WORKSHEET_LAYOUT_OPTIONS
+        self.layout_displaylist = getWorksheetLayouts()
```

Comment:
Careful when doing adapter/utility lookups inside the `__init__` method. The Zope Component Architecture (ZCA) might not be ready at this point. Also permission lookups will probably fail. You should add this as method or property.

Reply:
done